src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -142,7 +148,7 @@ public boolean isParticipating() {
     public List<BusResult> createBusResults() {
         var bus = getBus();
         if (breakers) {
-            return List.of(new BusResult(getVoltageLevelId(), bus.getId(), v, Math.toDegrees(angle)));
+            return List.of(new BusResult(getVoltageLevelId(), bbsIds != null ? bbsIds.toString() : bus.getId(), v, Math.toDegrees(angle)));
```

Comment:
I think we should probably output a bus result per bus bar section instead of a single one with a list serialized as bus id.

Reply:
Yes, if @geofjamg agrees!